packages/framer-motion/src/value/use-spring.ts

```diff
@@ -56,7 +60,7 @@ export function useSpring(
             })
 
             return value.get()
-        })
+        }, stopAnimation)
```

Comment:
Was this something that used to be part of this `useMemo` but was captured incorrectly?